src/Sweden/Mews.Fiscalizations.Sweden/DTOs/CreateActivationRequest.cs

```diff
@@ -0,0 +1,104 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Mews.Fiscalizations.Sweden.DTOs;
+
+internal sealed class CreateActivationRequest
+{
+    [Required]
+    [JsonPropertyName("country")]
+    public required string CountryCode { get; init; }
+
+    [Required]
+    [JsonPropertyName("corporateId")]
+    public required string CorporateId { get; init; }
+
+    [Required]
+    [JsonPropertyName("cashRegisterName")]
+    public required string CashRegisterName { get; init; }
+
+    [JsonPropertyName("features")]
+    public string[] Features { get; init; }
+
+    [Required]
+    [JsonPropertyName("controlUnitSerial")]
+    public required string ControlUnitSerial { get; init; }
+
+    [Required]
+    [JsonPropertyName("controlUnitLocation")]
+    public required string ControlUnitLocation { get; init; }
+
+    [JsonPropertyName("validFrom")]
+    public long? ValidFromUnix { get; init; }
+
+    [JsonPropertyName("validTo")]
+    public long? ValidToUnix { get; init; }
+
+    [JsonPropertyName("registrationGeolocation")]
+    public Address Address { get; init; }
+
```

Comment:
Checking the postman collection, there seems to be a few fields not included in this DTO like: 
"controlUnitGeolocation", "controlUnitSerial", "connectionCode" and "comment". Is it because they are not needed?

Reply:
`controlUnitSerial` should be null if controlUnitLocation is remote and i believe that's our case.
no idea what `connectionCode `is and its optional so we won't provide it


`controlUnitGeolocation` based on the docs, if we provide the `registrationGeolocation` (which we do), then its content will be copied to `controlUnitGeolocation`